docs/Customization/Keys-Traits-Configs.rst

```diff
@@ -42,64 +42,37 @@ Top-level traits should be defined and documented in subprojects, alongside thei
 
 Below we see the traits for the GCD example. The Lazy trait connects the GCD module to the Diplomacy graph, while the Implementation trait causes the ``Top`` to instantiate an additional port and concretely connect it to the GCD module.
 
-.. literalinclude:: ../../generators/example/src/main/scala/GCD.scala
+.. literalinclude:: ../../generators/chipyard/src/main/scala/GCD.scala
     :language: scala
     :start-after: DOC include start: GCD lazy trait
     :end-before: DOC include end: GCD imp trait
 
 These traits are added to the default ``Top`` in Chipyard.
 
-.. literalinclude:: ../../generators/example/src/main/scala/Top.scala
+.. literalinclude:: ../../generators/chipyard/src/main/scala/Top.scala
     :language: scala
     :start-after: DOC include start: Top
     :end-before: DOC include end: Top
 
 Mixins
 ------
 
-Mixins set the keys to a non-default value. Together, the collection of Mixins which define a configuration generate the values for all the keys used by the generator.
+Config mixins set the keys to a non-default value. Together, the collection of Mixins which define a configuration generate the values for all the keys used by the generator.
 
 For example, the ``WithGCDMixin`` is parameterized by the type of GCD widget you want to instantiate. When this mixin is added to a config, the ``GCDKey`` is set to a instance of ``GCDParams``, informing the previously mentioned traits to instantiate and connect the GCD widget appropriately.
 
-.. literalinclude:: ../../generators/example/src/main/scala/ConfigMixins.scala
+.. literalinclude:: ../../generators/chipyard/src/main/scala/GCD.scala
     :language: scala
     :start-after: DOC include start: GCD mixin
     :end-before: DOC include end: GCD mixin
 
 We can use this mixin when composing our configs.
 
-.. literalinclude:: ../../generators/example/src/main/scala/RocketConfigs.scala
+.. literalinclude:: ../../generators/chipyard/src/main/scala/RocketConfigs.scala
     :language: scala
     :start-after: DOC include start: GCDTLRocketConfig
     :end-before: DOC include end: GCDTLRocketConfig
 
-
-BuildTop
```

Comment:
`BuildTop` doesn't exist anymore?

Reply:
The `BuildTop` composition via recursive functions (which this section describes) does not exist anymore.